common/modules/farm/farm_position/src/farm_position.rs

```diff
@@ -0,0 +1,105 @@
+#![no_std]
+
+multiversx_sc::imports!();
+multiversx_sc::derive_imports!();
+
+use common_structs::FarmTokenAttributes;
+
+#[multiversx_sc::module]
+pub trait FarmPositionModule:
+    farm_token::FarmTokenModule
+    + utils::UtilsModule
+    + permissions_module::PermissionsModule
+    + multiversx_sc_modules::default_issue_callbacks::DefaultIssueCallbacksModule
+{
+    #[payable("*")]
+    #[endpoint(updateTotalFarmPosition)]
```

Comment:
I think we have to integrate this module into metastaking and we can delete the merge of positions from staking and farming in the boosted metastaking module.

Reply:
fixed